rssfeed/templatetags/rssfeed_tags.py

```diff
@@ -0,0 +1,11 @@
+from django import template
+
+from rssfeed.models import Entry
+
+register = template.Library()
+
+
+@register.inclusion_tag("rssfeed/inclusion_tags/render_rssfeed.html")
+def render_rssfeed(count=10):
+    entries = Entry.objects.all()
```

Comment:
@rijkstofberg I'm not sure I know what you mean by this? I'd prefer to keep as much of the processing in the task as possible, since the task is running on a separate thread.